src/neptune/internal/utils/uncaught_exception_handler.py

```diff
@@ -21,59 +21,59 @@
 import uuid
```

Comment:
Shouldn't we remove an entry from `pyproject.toml` after applying mypy suggestions?